Gruntfile.coffee

```diff
@@ -301,10 +301,10 @@ module.exports = (grunt) ->
 
 	grunt.util.linefeed = "\n"
 	# Project configuration.
-	grunt.initConfig
+	@initConfig
 
 		# Metadata.
-		pkg: grunt.file.readJSON("package.json")
+		pkg: @file.readJSON("package.json")
```

Comment:
That's fine, then it should be done consistently and pushed down to the themes too